index.js

```diff
@@ -3,15 +3,20 @@ var AlexaUtterances = require("alexa-utterances");
 var SSML = require("./to-ssml");
 var alexa = {};
 
-alexa.response = function() {
+alexa.response = function(hasSession) {
```

Comment:
Maybe we need to move `session` outside of `request` and `response` since it's an independent object not related to request or response instantly. 
I'd even say it's between them.
So we can provide it as an alone object: 
```
app.pre = function(request, response, type, session) {
  ...
};

app.intent("sampleIntent", {
    "slots": {...},
    "utterances": [...]
  },
  function(request, response, session) {
    ...
  }
);
```
And call `session` methods from existed methods of `request` and `response` for backwards compatibility.